### PR TITLE
Added support for running the backend with just docker-compose up

### PR DESCRIPTION
**Pull request by jalotra**
Now docker is used for everything, 

build : 
```
docker compose -f docker-compose.yml build
```

run : 
```
docker compose -f docker-compose.yml up 
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .PHONY: build run
 
-BIN := bin/gullak.bin
+BIN := /usr/local/bin/gullak.bin
```
Comment (mr-karan) [inline Makefile:3]: Can this be `bin/` ? `/usr/local` will pollute the host env.

**Review by mr-karan — CHANGES_REQUESTED** [review]
Thanks. Left a minor comment.

File: Makefile (hunk 1 above)
Comment (jalotra) [reply Makefile]: done, path back to `$LOCAL_DIR/bin/..`

**Comment by mr-karan** [comment]
Thank you!

**Pull request merged**